reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-70.md

```diff
@@ -0,0 +1,809 @@
+Core 3.1 brings back a host of .NET Framework APIs (especially on Windows), enabling significantly
+more backwards compatibility with existing Windows PowerShell modules. This includes many modules on
+Windows that require GUI functionality like `Out-GridView` and `Show-Command`, as well as many role
+management modules that ship as part of Windows.
+
+On Windows, we've also added a **UseWindowsPowerShell** switch to `Import-Module` to ease the
+transition to PowerShell 7 for those using incompatible modules. This switch creates a proxy module
+in PowerShell 7 that uses a local Windows PowerShell process to implicitly run any cmdlets contained
+in that module. For more information on this functionality, check out the [Import-Module](TODO)
+documentation.
+
+For more info, check out our module compatibility table showing off the latest up-to-date modules
+that work with PowerShell 7: [Module Compatibility Table](TODO)
+
+## Improved ForEach-Object with parallelism support
+
+The `ForEach-Object` cmdlet, which iterates items in a collection, now has built-in parallelism with
+the new **Parallel** parameter.
+
+Beginning with PowerShell 7.0, a third parameter set is available that runs each script block in
+parallel. There is a **ThrottleLimit** parameter that limits the number of parallel script blocks
+running at a time. As before, use the **$_** variable to represent the current input object in the
+script block. Use the **$using:** scope to pass variable references to the running script block.
+
+By default, parallel script blocks use the current working directory of the caller that started the
+parallel tasks.
+
+This example retrieves 50,000 log entries from 5 system logs on a local Windows machine.
+
+```powershell
+$logNames = 'Security','Application','System','Windows PowerShell','Microsoft-Windows-Store/Operational'
+
+$logEntries = $logNames | ForEach-Object -Parallel {
+    Get-WinEvent -LogName $_ -MaxEvents 10000
+} -ThrottleLimit 5
+
+$logEntries.Count
+
+50000
+```
+
+The **Parallel** parameter specifies the script block that is run in parallel for each input log name.
+The **ThrottleLimit** parameter ensures that all five script blocks run at the same time.
+
+[!NOTE] The cmdlet `ForEach-Object` includes a switch parameter **AsJob** to run as a background
+job. For further details see;
+[Background Jobs](https://docs.microsoft.com/powershell/scripting/developer/cmdlet/background-jobs?view=powershell-7)
+
+For more information, see
+[ForEach-Object](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/foreach-object?view=powershell-7)
+
+## Ternary operator
+
+PowerShell 7.0 introduced a new syntax using the ternary operator which behaves like a simplified
+if-else statement. It follows the C# ternary operator syntax:
+
+```
+<condition> ? <if-true> : <if-false>
+```
+
+The ternary operator behaves like the simplified `if-else` statement. The condition-expression will
```

Comment:
"The ternary operator behaves like the simplified `if-else` statement."

This is a repeated statement from the first sentence.

Reply:
@alexandair Thanks for the help with the editing sir!